src/driver/mysql/MysqlDriver.ts

```diff
@@ -478,42 +478,50 @@ export class MysqlDriver implements Driver {
     /**
      * Prepares given value to a value to be persisted, based on its column type or metadata.
      */
-    prepareHydratedValue(value: any, columnMetadata: ColumnMetadata): any {
-        if (value === null || value === undefined)
-            return columnMetadata.transformer ? ApplyValueTransformers.transformFrom(columnMetadata.transformer, value) : value;
+    prepareHydratedValue(value: any, columnMetadata: ColumnMetadata | string): any {
+        const columnType: any = columnMetadata instanceof ColumnMetadata ? columnMetadata.type : columnMetadata;
```

Comment:
I would revert this statement to `typeof columnMetadata === "string" ? x : y` because `instanceof` has proven to be unreliable in some environments.